update-images.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+set -e
+
+[[ -z "${CLIENT_ID}" ]] && echo "CLIENT_ID not set" &&  exit 1;
+[[ -z "${CLIENT_SECRET}" ]] && echo "CLIENT_SECRET not set" &&  exit 1;
+
+TOKEN_ENDPOINT=https://login.curity.io/oauth/v2/token
+RELEASE_API=https://releaseapi.curity.io/releases
+SCOPE="release_download release_read"
+
+LATEST_RELEASE=$(find -- * -maxdepth 0 -type d | sort -rh | head -n 1)
+
+while IFS= read -r VERSION
+do
+  echo "Downloading version ${VERSION}";
+  ACCESS_TOKEN=$(curl -d "grant_type=client_credentials&client_secret=${CLIENT_SECRET}&client_id=${CLIENT_ID}&scope=${SCOPE}" "${TOKEN_ENDPOINT}" | jq -r '.access_token')
```

Comment:
probably want to pass `-s -S` to curl since you're piping it. Otherwise, the progress bar will be output.

Reply:
also, you can do `-u ${CLIENT_ID}:${CLIENT_SECRET} -d grant_type=client_credentials -d scope=$SCOPE` if you want